src/components/Collections/Collection.tsx

```diff
@@ -440,7 +440,11 @@ export function Collection({
         <Meta
           title={`${collection.name} - collection posted by ${collection.user.username}`}
           description={collection.description ?? undefined}
-          deIndex={collection.read !== 'Public' ? 'noindex, nofollow' : undefined}
+          deIndex={
+            collection.read !== 'Public' || collection.availability === Availability.Unsearchable
```

Comment:
Not sure that this change is necessary.  `!== 'Public'` will handle the unsearchable case as well, right?

Reply:
I mean, yeah - but I guess we may want a collection to still be public + not searchable (i.e, you can get to it via links - similar to unlisted), so no harm in keeping this